internal/brokers/testdata/TestGetAuthenticationModes/golden/get_authentication_modes_and_generate_validators

```diff
@@ -0,0 +1,8 @@
+MODES:
+[{"id":"mode1","label":"Mode 1"}]
```

Comment:
I think here we can have one test case returning more than one mode. (needs changes in the broker). This aligns with my comment on the internal/services/pam test branch.